src/plugins/tabs/components/TabsComponent.vue

```diff
@@ -37,6 +37,7 @@
         v-for="(tab, index) in tabsList"
```

Comment:
think we could quickly implement the [ARIA tab role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role) here? We've already done this for the inspector tabs